README.md

```diff
@@ -40,6 +40,40 @@ The following command will restart all pods in the `web` and `jobs` deployments:
 
 `kubernetes-restart <kube namespace> <kube context> --deployments=web,jobs`
 
+### Deploying Kubernetes secrets
+
+Since their data is only base64 encoded, Kubernetes secrets should not be committed to your repository. Instead, `kubernetes-deploy` supports generating secrets from an encrypted [ejson](https://github.com/Shopify/ejson) file in your template directory. Here's how to use this feature:
+
+1. Install the ejson gem: `gem install ejson`
+2. Generate a new keypair: `ejson keygen` (prints the keypair to stdout)
+3. Create a Kubernetes secret in your target namespace with the new keypair: `kubectl create secret generic ejson-keys --from-literal=YOUR_PUBLIC_KEY=YOUR_PRIVATE_KEY --namespace=TARGET_NAMESPACE`
```

Comment:
I wrote these instructions from the perspective of someone trying to use `kubernetes-deploy` directly, without any of the magic from our platform. Our users wouldn't be able to do this even if they wanted to, since they don't have `kubectl` access. The draft docs for our users are in the Cloudplatform User Guide as suggestions. How about I add a line at the top of this section saying that Shopify employees using our Cloudplatform should consult the user guide instead?